src/index.jsx

```diff
@@ -42,6 +43,10 @@ const cleanup = render(
             <Router>
                 <Nav network={network} />
                 <Routes>
+                    <Route path="*">
+                        <Navigate href={"/404"} />
+                    </Route>
```

Comment:
```suggestion
                    <Route path="*" element={<Navigate href={"/404"} />} />
```

Else the initial load of the page navigates to the 404 (check Vercel deployment link if you don't believe me)

https://github.com/solidjs/solid-router/issues/132#issuecomment-1166318198